### PR TITLE
libsamplerate: bump to 0.2.1

### PR DESCRIPTION
**Pull request by enzo1982**
libsndfile and libsamplerate are now maintained by the libsndfile project. Unfortunately, this went largely unnoticed as the previous websites have not been updated to mention this.

This bumps the HaikuPorts recipe to the current libsamplerate 0.2.1 release. The new release no longer ships HTML docs, so the patch to adjust the HTML doc path is no longer needed.

### CODE REVIEW

**Comment by korli** [comment]
Would you mind switching to cmake? Maybe also a TEST section with "make test".
Otherwise we can also merge as is.

**Comment by enzo1982** [comment]
> Would you mind switching to cmake? Maybe also a TEST section with "make test".

Alright, I'll give cmake a try and see if I can add the TEST section.

**Comment by enzo1982** [comment]
Pushed the commit to switch to cmake and add the TEST section.

File: media-libs/libsamplerate/libsamplerate-0.2.1.recipe [hunk 1]
```diff
@@ -41,29 +40,29 @@ BUILD_REQUIRES="
...
-	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
 BUILD()
 {
-	autoreconf -fi
-	runConfigure ./configure
+	cmake -DCMAKE_INSTALL_PREFIX=$prefix \
```
Comment (korli) [inline media-libs/libsamplerate/libsamplerate-0.2.1.recipe:52]: Please try to use $cmakeDirArgs

**Comment by enzo1982** [comment]
> Please try to use $cmakeDirArgs

Done.

**Pull request merged**